cmd/queue/main.go

```diff
@@ -500,14 +511,7 @@ func supportsMetrics(env config, logger *zap.SugaredLogger) bool {
 
 func buildAdminServer(healthState *health.State, probe *readiness.Probe, logger *zap.SugaredLogger) *http.Server {
 	adminMux := http.NewServeMux()
-	adminMux.HandleFunc(requestQueueHealthPath, healthState.HealthHandler(func() bool {
-		if !probe.ProbeContainer() {
-			return false
-		}
-		logger.Info("User-container successfully probed.")
```

Comment:
if we're cool dropping this, do we still need the `logger`?

Reply:
Good finding. Done.